javascript/javascript1/week1/ez-namey.js

```diff
@@ -0,0 +1,5 @@
+const firstWords = ['Word', 'Fresh', 'Brand', 'Name', 'Quick', 'Innovative', 'Domain', 'Brand', 'One', 'Biz'];
+const secondWords = ['Lab', 'Books', 'Root', 'Station', 'Solutions', 'Creations', 'Puzzler', 'Bucket', 'Click', 'Namez'];
+const randomNumber = Math.floor(Math.random() * 10) + 0;
```

Comment:
```suggestion
const randomNumber = Math.random() * firstWords .length
```
I think this should work also...what do you think . Also it's usually a good practice to not use constants in expressions

Reply:
Modified the expression as you mentioned and used let instead of const